packages/cli/yaml/yaml-schema/src/schemas/TypeReferenceSchema.ts

```diff
@@ -34,3 +34,12 @@ export const TypeReferenceDeclarationWithEnvOverride = extendTypeReferenceSchema
     }).shape
 );
 export type TypeReferenceDeclarationWithEnvOverride = z.infer<typeof TypeReferenceDeclarationWithEnvOverride>;
+
+export const TypeReferenceDeclarationWithContentType = extendTypeReferenceSchema(
+    DeclarationWithNameSchema.extend({
+        ["content-type"]: z.optional(z.string(), {
```

Comment:
@abvthecity Is this as simple as a string value that the generator needs to append?

Reply:
not exactly.
the `content-type` here can be a list of acceptable content-types, i.e. `image/png, image/jpeg`.

the sdk should be able to infer the content-type from whatever file is being provided.

for string values (application/json), however, yes.